releasenotes/notes/disallow_num_ctrl_qubits_zero-eb102aa087f30250.yaml

```diff
@@ -0,0 +1,8 @@
+---
+fixes:
+  - |
+  Previously it was possible to set the number of control qubits to zero in which case the
+  the original, potentially non-controlled, operation would be returned. This could cause
+  an AttributeError if the caller attempts to access an attribute which only ControlledGates
+  have. This fix adds a getter/setter for num_ctrl_qubits to handle validation.
+    
```

Comment:
```suggestion
    Previously it was possible to set the number of control qubits to zero in which case the
    the original, potentially non-controlled, operation would be returned. This could cause
    an AttributeError if the caller attempts to access an attribute which only ControlledGates
    have. This fix adds a getter/setter for num_ctrl_qubits to handle validation.
```